# place config files in the home directory. The game directory is readonly.
Index: wyrmgus-2.0.0/CMakeLists.txt
===================================================================
--- wyrmgus-2.0.0.orig/CMakeLists.txt	2016-03-20 20:34:21.856602766 +0100
+++ wyrmgus-2.0.0/CMakeLists.txt	2016-03-20 20:34:21.856602766 +0100
@@ -785,7 +785,7 @@
 option(ENABLE_DEV "Install Stratagus game development headers files" OFF)
 option(ENABLE_UPX "Compress Stratagus executable binary with UPX packer" OFF)
 option(ENABLE_STRIP "Strip all symbols from executables" OFF)
-option(ENABLE_USEGAMEDIR "Place all files created by Stratagus(logs, savegames) in game directory(old behavior), otherwise place everything in user directory(new behavior)" ON)
+option(ENABLE_USEGAMEDIR "Place all files created by Stratagus(logs, savegames) in game directory(old behavior), otherwise place everything in user directory(new behavior)" OFF)
 option(ENABLE_MULTIBUILD "Compile Stratagus on all CPU cores simltaneously in MSVC" ON)
 
 if(NOT WITH_RENDERER)
